src/status_im/multiaccounts/login/core.cljs

```diff
@@ -170,6 +170,7 @@
               (stickers/init-stickers-packs)
               (mobile-network/on-network-status-change)
               (chaos-mode/check-chaos-mode)
+              (status/validate-mnemonic "abacus")
```

Comment:
is that some leftover?

Reply:
i think PR just WIP

Reply:
Correct, this was just to make sure the new function fires every time I login. I need some guidance on how to call it from `status-react` since I'm getting undefined errors when I call it at present.

Reply:
Who's the right person to ask on this?  I'm still not clear on if I'm missing some function declaration somewhere in the status-react side to be able to call the new `validate-mnemonic` function or if I am missing something on the status-go side.